ui/components/app/alerts-system/alert-modal/alert-modal.tsx

```diff
@@ -88,97 +104,133 @@ export function getSeverityStyle(severity?: Severity) {
       };
     default:
       return {
-        background: BackgroundColor.infoMuted,
+        background: BackgroundColor.backgroundDefault,
         icon: IconColor.infoDefault,
       };
   }
 }
 
-function AlertHeader({
-  selectedAlert,
-  customAlertTitle,
+export function AlertProvider({
```

Comment:
As this is used externally as part of the `SecurityAlertBanner`, then should this be defined in its own file and folder separate to `alert-modal`?

Reply:
yes, I created  `AlertProvider` component and removed it from alert-modal.